### ISSUE TITLE
Text for semester on thumbnail pages doesn't have enough contrast for WCAG AAA

### ISSUE THREAD

**Issue by benoitlord** [benoitlord/portfolio_nuxt#3]
The text for the semester on thumbnail pages (#555 on #ddd) doesn't pass contrast for WCAG AAA.
